### PR TITLE
OKTA-544755: Add client application as a member in Custom Role assignment operations

### PR DESCRIPTION
**Pull request by alimohammadi-okta**
## Description:
- What's changed? The client application can be used as a **_member_** in Custom Role assignment operations.
- Is this PR related to a Monolith release? Yes. 2023.05.0

### Resolves:

* [OKTA-544755](https://oktainc.atlassian.net/browse/OKTA-544755)

### Screen recording
https://user-images.githubusercontent.com/110849349/207905002-83c2cd19-fcfd-4eeb-b492-b3d71f8d1fba.mov
## Reviewer(s):
@alifatolahi-okta @vasishtajayanti-okta @susanharper-okta

### CODE REVIEW

**Review by vasishtajayanti-okta — APPROVED** [review]

**Review by susanharper-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md
Comment (alifatolahi-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:179]: `ea` -> `beta`

File: packages/@okta/vuepress-site/docs/reference/api/roles/index.md
Comment (alifatolahi-okta) [inline packages/@okta/vuepress-site/docs/reference/api/roles/index.md:1018]: `beta`

File: packages/@okta/vuepress-site/docs/reference/api/roles/index.md
Comment (alifatolahi-okta) [inline packages/@okta/vuepress-site/docs/reference/api/roles/index.md:1106]: `beta`

File: packages/@okta/vuepress-site/docs/reference/api/roles/index.md
Comment (alifatolahi-okta) [inline packages/@okta/vuepress-site/docs/reference/api/roles/index.md:1200]: `beta`

File: packages/@okta/vuepress-site/docs/reference/api/roles/index.md
Comment (alifatolahi-okta) [inline packages/@okta/vuepress-site/docs/reference/api/roles/index.md:3543]: `beta`

**Review by alifatolahi-okta — APPROVED** [review]
LGTM, we'll have to wait for QA and Fed sign-off, which may happen in Jan

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Review by susanharper-okta — APPROVED** [review]
LGTM 🚢

**Comment by brentschaus-okta** [comment]
Will add additional concept content.

**Review by brentschaus-okta — APPROVED** [review]
LGTM

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md](https://github.com/okta/okta-developer-docs/blob/b75ec9ac6d49353851aa2a1725523b93852f94d5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Frole-assignment%2Findex.md) | [73](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c7b789ce-d61b-4c26-a797-94336a475010) | 74  | 79  | 61  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/api/roles/index.md](https://github.com/okta/okta-developer-docs/blob/b75ec9ac6d49353851aa2a1725523b93852f94d5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Froles%2Findex.md) | [68](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f4250b68-a368-433a-b82d-02e6d5259ec1) | 82  | 53  | 62  | 74  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.b6a15f0d-99e6-4607-87bf-18fd914eef84?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Pull request merged**
